### DISCUSSION TITLE
Using in a project without composer impossible?

### DISCUSSION THREAD

**Discussion by olek07** [sourcebroker/t3api#40] [Q&A] [answered]
Hi!

The zip-file https://extensions.typo3.org/extension/download/t3api/1.2.1/zip is shipped without dependencies. The vendor folder is absent. How were it possible to use the extension in a T3 8.7 project, that was installed WITHOUT composer?

**Comment by kszymukowicz** [comment]
We always used t3api only with composer based projects.

**Comment by klamparski** [comment, marked as answer]
@olek07 We do not plan to support installation without composer but you can create a fork with all the vendors inside extension. Here is an example: https://github.com/sourcebroker/t3api/issues/15#issuecomment-715894988

**Reply by olek07** [reply]
I could install the extension without composer
